Ng favourites

Nikolay-Gavazov opened the pull request:
Import all from event page and chage to work in favourites.
Create get favourites function to take the user favourites.

Nikolay-Gavazov committed:
import all from event page and chage to work in favourites

Nikolay-Gavazov committed:
create get favourites function to take the user favourites

Getsov commented on src/app/notifications/notifications.page.html:
```diff
@@ -1,14 +1,23 @@
+    ></app-search>
+  </div>
+  <app-title-fades [color]="titleColor" [title]="titleText" />
+  <div class="ion-padding-horizontal" *ngIf="(user?.role == 'admin') || (user?.role == 'organizer')">
```
What is this div supposed to do?

Nikolay-Gavazov replied:
<div class="page-title">

Nikolay-Gavazov replied:
"<div class="page-title">"

Nikolay-Gavazov replied:
div class="page-title"

Getsov commented on src/app/notifications/notifications.page.html:
```diff
@@ -1,14 +1,23 @@
+      [backButton]="backButton"
+      [defaultHref]="defaultHref"
+    ></app-header>
+    <app-search
```
Do we have also the filters?

Nikolay-Gavazov replied:
There will be, just need to make a function on the backend, because now I checked that there is not for the moment

Nikolay-Gavazov committed:
Merge branch 'dev' into NG-Favourites

Nikolay-Gavazov committed:
add parrent

So I will submit which parent is using this component so that the correct request can be made.

Nikolay-Gavazov committed:
Merge branch 'dev' into NG-Favourites

Nikolay-Gavazov committed:
Move search component from events to shared

Nikolay-Gavazov committed:
change name to favourites

Nikolay-Gavazov committed:
just formatting

Nikolay-Gavazov committed:
add query to the favourites service function
